monero-outreach-docs/translations/es/storyboard/patents_es.md

```diff
@@ -0,0 +1,95 @@
+
+Eventualmente la solicitud quizás sea publicada en la web del USPTO (uspto.gov). La publicación llega generalmente 18 meses después de la fecha más temprana de la solicitud de protección de patente, llamada la fecha prioritaria. La fecha de prioridad suele ser la fecha en que se presenta la solicitud, pero puede ser la fecha en que se presentó una solicitud provisional relacionada. Los inventores pueden presentar una solicitud de no publicación en el momento de la presentación; sin embargo, incluso en este caso, la solicitud de patente eventualmente se publicará si se otorga.
+
+Una patente estadounidense sólo ofrece protección en los Estados Unidos. Muchas países (152) son parte del Tratado de Cooperación de Patentes (PCT) y tienen un proceso con algunas conexiones y similitudes, pero en general para tener protección en cualquier país requiere presentarla a través de la oficina de patentes de ese país. Si un solicitante de EE. UU. Presenta una solicitud especial, llamada solicitud internacional (o PCT), dentro de un año de la presentación en los EE. UU., ese solicitante puede tener un período de tiempo (por ejemplo, hasta 30 meses) a partir de la fecha de presentación de los EE. UU. para presentar en los otros países que firmaron el tratado. Las presentaciones directas en oficinas de patentes extranjeras también son posibles y son comunes en países fuera del PCT.
+### _Cumplimiento de Patentes_
+
+Una vez que una empresa como CipherTrace ha obtenido una patente, tiene que llevar a cabo su propia aplicación mediante licencias o litigios. No hay policía ni otras autoridades gubernamentales. Si una tecnología está cubierta por una patente de EE. UU., Otros no pueden sin permiso, en los Estados Unidos, fabricar, usar, vender, ofrecer vender o importar esa tecnología. La aplicación comienza con el conocimiento de la actividad infractora (que puede ser difícil para el software que se ejecuta detrás de un firewall, a veces incluso para el software distribuido como binarios). Después del descubrimiento, la empresa se pondría en contacto con la entidad potencialmente infractora y le pediría que licenciara la tecnología o dejara de usarla. Si no lo hace, la empresa puede iniciar un litigio para detener la infracción.
+
+A menudo, es mejor evitar los litigios, ya que el costo promedio de los litigios de patentes es de millones de dólares [8]. Además, el propietario de una patente que presenta una demanda corre un riesgo significativo porque el demandado puede localizar tecnologías y publicaciones preexistentes que invalidan la patente. Si tal esfuerzo tiene éxito, el propietario de la patente puede quedarse con grandes facturas legales y una patente inválida.
```

Comment:
A menudo, es mejor evitar los litigios, ya que el costo promedio de los litigios de patentes es de millones de dólares [[**8**](https://www.monerooutreach.org/stories/monero-patents.html#b8)]. Además, el propietario de una patente que presenta una demanda corre un riesgo significativo porque el demandado puede localizar tecnologías y publicaciones preexistentes que invalidan la patente. Si tal esfuerzo tiene éxito, el propietario de la patente puede quedarse con grandes facturas legales y una patente inválida.

Missing the link for 8 -> https://www.monerooutreach.org/stories/monero-patents.html#b8